src/components/DatePicker/index.android.js

```diff
@@ -8,87 +8,76 @@ import CONST from '../../CONST';
 import {propTypes, defaultProps} from './datepickerPropTypes';
 import styles from '../../styles/styles';
 
-class DatePicker extends React.Component {
-    constructor(props) {
-        super(props);
-
-        this.state = {
-            isPickerVisible: false,
-        };
-
-        this.showPicker = this.showPicker.bind(this);
-        this.setDate = this.setDate.bind(this);
-    }
+function DatePicker({value, defaultValue, label, placeholder, errorText, containerStyles, disabled, maxDate, minDate, innerRef, onBlur, onInputChange}) {
+    const [isPickerVisible, setIsPickerVisible] = useState(false);
 
     /**
      * @param {Event} event
      * @param {Date} selectedDate
      */
-    setDate(event, selectedDate) {
-        this.setState({isPickerVisible: false});
+    function setDate(event, selectedDate) {
```

Comment:
I guess better replace the function with constant arrow for better consistency with the codebase